site2/docs/security-token-admin.md

```diff
@@ -8,38 +8,37 @@ sidebar_label: Token authentication admin
 
 Pulsar supports authenticating clients using security tokens that are based on [JSON Web Tokens](https://jwt.io/introduction/) ([RFC-7519](https://tools.ietf.org/html/rfc7519)).
 
-You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that
-is permitted to do some actions (for example, publish to a topic or consume from a topic).
+You can use tokens to identify a Pulsar client and associate with some "principal" (or "role") that is permitted to do some actions (for example, publish to a topic or consume from a topic).
 
-A user typically gets a user a token string from the administrator (or some automated service).
+A user typically gives a user a token string from the administrator (or some automated service).
 
 The compact representation of a signed JWT is a string that looks like as the follwing:
 
 ```
 eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJKb2UifQ.ipevRNuRP6HflG8cFKnmUPtypruRC4fb1DWtoLL62SY
 ```
 
-Application specifies the token when you are creating the client instance. An alternative is to pass a "token supplier" (a function that returns the token when the client library needs one).
+Application does the token specification when you create the client instance. An alternative is to pass a "token supplier" (a function that returns the token when the client library needs one).
```

Comment:
re-write this sentence. 
The token is specified by application when you create a client instance.